services/app/lib/codebattle_web/templates/page/landing.html.slim

```diff
@@ -1,49 +1,39 @@
-section.jumbotron
+section.jumbotron.bg-transparent.mb-0
   .container
     .row
-      .col-12.col-sm-6.col-xl-7.text-center.mb-5.mb-sm-0
-        h1.jumbotron-heading Achieve mastery through challenge
-          p.lead.text-muted Improve your skills by training with others on real code challenges
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-success.btn-lg = "Start playing"
-      .col-12.col-sm-6.col-xl-5.d-flex.align-content-center
+      .col-12.text-center.mb-4
+        h1.jumbotron-heading.mb-3 Achieve mastery through challenge
+        p.lead.mb-3 Improve your skills by training with others on real code challenges
+        a.btn.btn-success.btn-lg href="#{auth_path(@conn, :request, :github)}" = "Start playing"
+      .col-12.d-flex.align-content-center
         .d-flex.flex-wrap.w-100.justify-content-center.align-content-center
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-clojure.pr-1
-              = "Clojure"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-elixir.pr-1
-              = "Elixir"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-haskell.pr-1
-              = "Haskell"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-javascript-alt.pr-1
-              = "JavaScript"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-perl.pr-1
-              = "Perl"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-php.pr-1
-              = "PHP"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-python.pr-1
-              = "Python"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-ruby.pr-1
-              = "Ruby"
+          a.m-2.btn.btn-sm.btn-outline-secondary href="#{auth_path(@conn, :request, :github)}"
+            i.icon-clojure.pr-1
+            = "Clojure"
```

Comment:
Would be better to use loop for this, don't you think?

Reply:
I use copypasta pattern

Reply:
maybe latter, i will refactor this

Reply:
primi po bratski, brat